src/Pages/Login-Page/Login.jsx

```diff
@@ -0,0 +1,82 @@
+import { Nav } from "../../components/Nav/Nav";
+import { Header } from "../../components/Header/Header";
+import { Footer } from "../../components/Footer/Footer";
+import { NavForTab } from "../../components/NavForTab/NavForTab";
+import { NavLink } from "react-router-dom";
+import "./login.css";
+import "./loginResponsive.css";
+export function Login() {
+  return (
+    <div className="login-page">
+      <Nav />
+      <Header />
+      <main className="login__main">
+        <section className="login">
+          <form className="login-form">
+            <h1 className="login__title">Login</h1>
+            <div className="login-form--primary-container">
+              <label
+                className="login-form__label login-form--username"
+                htmlFor="userName"
+              >
+                <b>UserName</b>
+              </label>
+              <input
+                className="login-form__input-demo login-form__input--username"
+                type="email"
+                placeholder="Enter UserName"
+                name="uname"
+                required
+              />
+              <label
+                className="login-form__label login-form--password"
```

Comment:
It is better to keep the name structure consistent. You are using login-form__label and login-form--password. 
Replace login-form--password to login-form__password as this is more consistent with your other form classnames